From 8ad52b2e3d70ca39c794cf1e37c7f818a6460f93 Mon Sep 17 00:00:00 2001
From: Fabien Spindler <Fabien.Spindler@inria.fr>
Date: Fri, 6 Dec 2013 17:52:57 +0100
Subject: [PATCH 1/4] Added install rule for package.xml

---
 CMakeLists.txt | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index f83aaf5..75a2db5 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -2124,6 +2124,11 @@ ELSE()
 ENDIF()
 
 #----------------------------------------------------------------------
+# Install catkin package.xml
+install(FILES package.xml DESTINATION share/visp)
+#----------------------------------------------------------------------
+
+#----------------------------------------------------------------------
 # For Dart server and tests 
 # We use CDash set through CTestConfig.cmake file
 # Dashboards are sent to http://cdash.irisa.fr/CDash/index.php?project=ViSP
-- 
1.8.1.2

