From 75b059cc9654a46f9e355a806be8b75fe3b2e04c Mon Sep 17 00:00:00 2001
From: Fabien Spindler <Fabien.Spindler@inria.fr>
Date: Fri, 22 Nov 2013 14:55:00 +0100
Subject: [PATCH 2/3] Fix -lm library

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5f316e7..1735744 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -691,7 +691,7 @@ IF(USE_X11)
   SET(VISP_HAVE_X11 TRUE)  # for header vpConfig.h
   SET(VISP_HAVE_X11_FOUND "yes")  # for ViSP-third-party.txt
   LIST(APPEND VISP_EXTERN_INCLUDE_DIRS ${X11_INCLUDE_DIR})
-  LIST(APPEND VISP_EXTERN_LIBRARIES ${X11_LIBRARIES} -lm)
+  LIST(APPEND VISP_EXTERN_LIBRARIES ${X11_LIBRARIES} m)
   #MESSAGE("X11: ${X11_LIBRARIES}")
 ENDIF(USE_X11)
 
-- 
1.8.1.2

