From e775ffa20ff33fd0bdfb7cab8208083fb80e0d21 Mon Sep 17 00:00:00 2001
From: Fabien Spindler <Fabien.Spindler@inria.fr>
Date: Fri, 6 Dec 2013 17:53:18 +0100
Subject: [PATCH 2/4] Fix -lm library

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 75a2db5..6e24b9f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -691,7 +691,7 @@ IF(USE_X11)
   SET(VISP_HAVE_X11 TRUE)  # for header vpConfig.h
   SET(VISP_HAVE_X11_FOUND "yes")  # for ViSP-third-party.txt
   LIST(APPEND VISP_EXTERN_INCLUDE_DIRS ${X11_INCLUDE_DIR})
-  LIST(APPEND VISP_EXTERN_LIBRARIES ${X11_LIBRARIES} -lm)
+  LIST(APPEND VISP_EXTERN_LIBRARIES ${X11_LIBRARIES} m)
   #MESSAGE("X11: ${X11_LIBRARIES}")
 ENDIF(USE_X11)
 
-- 
1.8.1.2

