From c9313c4c758b804b7a409676494322d255425770 Mon Sep 17 00:00:00 2001
From: Fabien Spindler <Fabien.Spindler@inria.fr>
Date: Tue, 15 Apr 2014 14:17:07 +0200
Subject: [PATCH 2/2] Force CMAKE_INSTALL_LIBDIR=lib

---
 CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 30c81ac..8490dec 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -56,6 +56,7 @@ set(VISP_CMAKE_MODULE_PATH ${VISP_SOURCE_DIR}/CMakeModules)
 
 find_file(GNU_INSTALL_DIRS_FROM_CMAKE ${CMAKE_ROOT}/Modules/GNUInstallDirs.cmake)
 mark_as_advanced(GNU_INSTALL_DIRS_FROM_CMAKE)
+set(CMAKE_INSTALL_LIBDIR lib)
 if(GNU_INSTALL_DIRS_FROM_CMAKE)
   include(${CMAKE_ROOT}/Modules/GNUInstallDirs.cmake)
 else()
-- 
1.8.1.2

